### ISSUE TITLE
[Feature Request] Ability to "pause" or "save"

### ISSUE THREAD

**Issue by CatGamer23** [m417z/UWPSpy#8]
Essentially, I'd like a feature so we could either pause or save the current state, to browse through later. An example of how this could be used: Modifying the clipboard manager (win+v), which automatically closes when focusing on a different window, which in turn, closes UWPSpy. I want the mode to prevent the closing of UWPSpy, and just have it freeze on the last known state, so it can then be browsed through later.

It would be even better if it was possible to change something while paused, then have it apply the next time the app/thread is opened.

**Comment by m417z** [comment]
Using this mod to prevent the popup from closing on focus loss worked for me to some degree:
https://gist.github.com/m417z/69815de59bc1c2b724bde88054884f3c
It might take several attempts. Let me know if it works for you or if you need more details about setting it up.

**Comment by m417z** [comment]
Yes, it didn't work so well, but at least it allowed to inspect the popup eventually. Here's a better approach of a mod which prevents focus to be switched from UWPSpy, which allows to inspect some menus (e.g. the WinUI 3 context menus in Explorer) and popups (e.g. Win+V) (same link):
https://gist.github.com/m417z/69815de59bc1c2b724bde88054884f3c

> every borderthickness is set to 0 already, but visibly still set to 1. Not sure what thats about

I don't know. We have a channel in [the Windhawk Discord server](https://discord.com/servers/windhawk-923944342991818753) to discuss styles and the styler mods, perhaps you can get some help over there.

**Comment by m417z** [comment]
Yes, it has several flaws, eventually it's just a workaround. I think it's best to enable it, then get UWPSpy to open with the popup, then disable the mod. Then the message boxes will be able to get focus.
